taskcluster_util/taskcluster_download.py

```diff
@@ -65,8 +65,7 @@ def run(self):
         # check credentials file
         abs_credentials_path = os.path.abspath(self.options.credentials)
         if not os.path.isfile(abs_credentials_path):
-            logger.warning('{} is not a file or is not exist.'.format(abs_credentials_path))
-            exit(-1)
+            raise Exception('{} is not a file or is not exist.'.format(abs_credentials_path))
```

Comment:
After checking, this exception is already given for free by the `with` in Credentials.from_file. If I deleted this line and the one just above, I can get:

```2015-07-31 13:46:24,339 - __main__ 103 - ERROR - [Errno 21] Is a directory: '/home/jlorenzo/git/taskcluster-util-python/test-dir'```
`2015-07-31 13:44:38,635 - __main__ 105 - ERROR - [Errno 2] No such file or directory: '/home/jlorenzo/git/taskcluster-util-python/credential_non_existing.json'`